content/documentation/pages/6-recipes/7-scaling/0-index.md

```diff
@@ -0,0 +1,29 @@
+---
+path: 'recipes/scaling/'
+title: 'Scaling'
+description: 'Using Prometheus and Data Flow Scale API for autoscaling pipelines'
+summary: true
+---
+
+# Scale-Out Data Pipelines
+
+[Scalability](https://en.wikipedia.org/wiki/Scalability) is the ability of a system to cope with increased load by adding resources. The [vertical](https://en.wikipedia.org/wiki/Scalability#Vertical) scaling (up/down) adds resources such as CPUs, memory and storage to the `existing nodes`, while the [horizontal](https://en.wikipedia.org/wiki/Scalability#Horizontal) scaling (out/in) adds `additional nodes` to the system.
```

Comment:
> Let's start with a general definition of scalability, the types, and why it is useful.

Reply:
No need to highlight `existing nodes` and `additional nodes` differently — they are just literal terminology.